src/components/Responsive/Grid/Grid.tsx

```diff
@@ -1,10 +1,34 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactElement } from 'react';
 
 import UnresponsiveGrid from '../../Grid';
 import Context from '../Context';
 
-const Grid = ({ breakpoints, children, columns, gutter, ...passedProps }) => {
+type GridGutter = number | string;
```

Comment:
Super minor, but I'd usually put an empty newline before the multiline `type GridDefinition ...`

Not a big deal, but I like the consistency. It should be a pretty easy thing to eslint and autofix, if others agree.